compressible_flow/Transitional_Flat_Plate/LM/NLF/transitional_LM_model_ConfigFile.cfg

```diff
@@ -0,0 +1,320 @@
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%                                                                              %
+% SU2 configuration file                                                       %
+% Case description: LM model for NLF-0416 airfoil ___________________________  %
+% Author: Sunoh. Kang _______________________________________________________  %
+% Institution: Pusan National University ____________________________________  %
+% Date: 10. 24. 2022                                                           %
+% File Version 7.4.0 "Blackbird"                                               %
+%                                                                              %
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+
+% ------------- DIRECT, ADJOINT, AND LINEARIZED PROBLEM DEFINITION ------------%
+%
+% Solver type (EULER, NAVIER_STOKES, RANS,
+%              INC_EULER, INC_NAVIER_STOKES, INC_RANS,
+%              NEMO_EULER, NEMO_NAVIER_STOKES,
+%              FEM_EULER, FEM_NAVIER_STOKES, FEM_RANS, FEM_LES,
+%              HEAT_EQUATION_FVM, ELASTICITY)
+SOLVER= RANS 
+%
+% Specify turbulence model (NONE, SA, SST)
+KIND_TURB_MODEL= SST
+% Specify versions/corrections of the SST model (V2003m, V1994m, VORTICITY, KATO_LAUNDER, UQ, SUSTAINING)
+SST_OPTIONS= NONE
```

Comment:
Remove the comments please, especially any reference to possible options, which eventually become outdated.
Keep comments only on options that are essential for this tutorial.

Reply:
Okay,  I thought the other options aren't changed frequently. So, I remove the comments related to the turbulence and transition models.